system_test/aest_sync_SUITE.erl

```diff
@@ -21,7 +21,7 @@
     connect_node/3, disconnect_node/3,
     http_get/5,
     request/4,
-    wait_for_height/4,
```

Comment:
My personal preference would be to have two functions, `wait_for_height` and `wait_for_balance` even though internally it could be the same function. This way the tests would be less noisy, but this is personal and this looks fine.